synbols/drawing.py

```diff
@@ -263,10 +263,12 @@ class ImagePattern(RandomPattern):
         rotation: float, Maximum random rotation in radian, default 0.
         translation: float, Maximum random translation in proportion, default 1.
         crop: bool, Whether to take a random crop of the image or not, default True.
+        min_crop_size: float, Crop's minimal proportion from the image, default 0.2.
         seed : Optional[int], Random seed to use for transformation, default to None
     """
 
-    def __init__(self, root='/images', rotation=0, translation=0., crop=True, seed=None):
```

Comment:
We're using line lengths of 120 in flake8, so technically you wouldn't have to break this into two lines.